index.js

```diff
@@ -5,6 +5,9 @@ const removeBtn = document.querySelector('.remove-btn');
 const mobileNav = document.querySelector('#mobile-navigation');
 const mobileNavLinks = document.querySelectorAll('.mobile-item');
 const popupContainer = document.querySelector('.popup');
+const work = document.querySelector('.work-container');
+const about = document.querySelector('#about');
+
 
 const navToggle = () => {
   mobileNav.classList.toggle('active');
```

Comment:
- [ ] [OPTIONAL] Your modal doesn't show everything that's on it. please fix it. Perhaps use a vertical overflow to allow a scroll.

|  | |
| ------------- | ------------- |
| Your Porfolio | <img width="761" alt="image" src="https://user-images.githubusercontent.com/52934128/223513010-91a9bddc-c325-4eb4-a092-0907a4509db9.png">  |
| Template Design | <img width="577" alt="image" src="https://user-images.githubusercontent.com/52934128/223513548-2f27687f-8fde-46cf-9454-463932153f70.png">  |